.github/workflows/test.yml

```
@@ -10,20 +10,63 @@ on:
+          git clone https://github.com/SimonKagstrom/kcov.git && \
+          cd kcov && git checkout ${KCOV_VERSION} && mkdir build && cd build && \
+          cmake .. && make && sudo make install && \
+          kcov --version
+
+      - name: Run tests with code coverage
+        run: kcov --include-pattern=.sh,.bash --bash-parse-files-in-dir=. --exclude-pattern=vendor,node_modules,.coverage-html,tests "$(pwd)"/.coverage-html .node_modules/.bin/bats tests
+        shell: bash
+        continue-on-error: ${{ vars.CI_TEST_IGNORE_FAILURE == '1' }}
+
+      - name: Upload coverage report as an artifact
+        if: matrix.os == 'ubuntu-latest'
```

actionlint (1.7.7)

property "os" is not defined in object type {}

(expression)